test/stdlib/Runtime.swift.gyb

```diff
@@ -556,7 +556,7 @@ Runtime.test("Struct layout with reference storage types") {
 Runtime.test("SwiftError layout constants for LLDB") {
 #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
   let RTLD_DEFAULT = UnsafeMutableRawPointer(bitPattern: -2)
-#elseif os(Linux)
+#elseif os(Linux) || os(Android)
   let RTLD_DEFAULT = UnsafeMutableRawPointer(bitPattern: 0)
```

Comment:
This is wrong.  bionic defines `RTLD_DEFAULT` as:

```
#if defined(__LP64__)
#define RTLD_DEFAULT  __BIONIC_CAST(reinterpret_cast, void*, 0)
#else
#define RTLD_DEFAULT  __BIONIC_CAST(reinterpret_cast, void*, 0xffffffff)
#endif
```

We need to emulate that behaviour.